doc/contributing.rst

```diff
@@ -158,5 +158,73 @@ Happy testing!
 Connecting new machine learning libraries
 =========================================
 
-Coming soon - please stay tuned!
+Content of the Library
+~~~~~~~~~~~~~~~~~~~~~~
 
+To get an understanding of what OpenML does and why support of the latest
+machine learning libraries can be beneficial for the community, please refer
+`here <https://openml.github.io/OpenML/>`_.
+
+Various `examples <https://openml.github.io/openml-python/develop/examples/index.html>`_
+can show how OpenML works in
+practice.
+Moreover `this example <https://openml.github.io/openml-python/develop/examples/
+flows_and_runs_tutorial.html>`_
+shows how scikit-learn currently works with OpenMl as an extension. The
```

Comment:
Do you mean OpenML-Python instead of OpenML here? Anyway, this should be OpenML instead of OpenMl (camelcase of the l).